requirements.txt

```diff
@@ -1,11 +1,2 @@
 -e git+https://github.com/nipy/nipype.git@master#egg=nipype
 -e git+https://github.com/incf/pybids.git@master#egg=pybids
-numpy
-lockfile
-future
-scikit-learn
-matplotlib
-nilearn
-sklearn
-nibabel
-niworkflows>=0.0.3a3
```

Comment:
Good reading, thanks. I think you are right. Why don't we do what this user suggested: https://www.reddit.com/r/Python/comments/3uzl2a/setuppy_requirementstxt_or_a_combination/cxka1a4

so both approaches will still work?